docs/getting-started/sdk/secrets-manager/integrations/kubernetes.mdx

```diff
@@ -0,0 +1,301 @@
+  Kubectl pointed to it as the current context for local development.
+
+</TabItem>
+</Tabs>
+
+You will also need:
+
+- A
+  [Bitwarden Organization with Secrets Manager](https://bitwarden.com/help/sign-up-for-secrets-manager/).
+  You will need the organization ID GUID for your organization.
+- An [access token](https://bitwarden.com/help/access-tokens/) for a Secrets Manager service account
+  tied to the projects you want to pull.
+
+## Setup and configuration
+
+Clone the repository:
+
+```bash
+git clone https://github.com/bitwarden/sm-kubernetes.git
+```
+
+Open Visual Studio Code to the
+[src/sm-operator](https://github.com/bitwarden/sm-kubernetes/tree/main/src/sm-operator) subfolder.
+Opening here will allow all of the Dev Container, Go language extensions, tasks, and launch settings
+to work correctly.
+
+<Tabs>
+<TabItem value="dev-container" label="Dev Container" default>
+
+Dev Container setup is automated. This will create a Kind cluster and setup all necessary software
+and SDK binaries.
+
+- Press `F1` to open the command palette
+- Start typing `Dev Containers: Reopen in Container`
+
+</TabItem>
+<TabItem value="direct" label="Direct Setup">
+
+:::info
+
+Please note that the Visual Studio Code debugger for Go does not work correctly if you open your
+workspace via a symlink anywhere in the path. For debugging to work, you should open it from the
+full path of the repository.
+
+:::
+
+- Install all requirements listed in the Direct Setup requirements
+- Create a Kind cluster using `kind create cluster`
+- Run `make setup` to download the appropriate Secrets Manager SDK binaries
+
+</TabItem>
+</Tabs>
+
+### Configuration settings
+
+A `.env` file will be created under
+[src/sm-operator](https://github.com/bitwarden/sm-kubernetes/tree/main/src/sm-operator) once the Dev
+Container is created or `make setup` has been run. The following environment variable settings can
+be updated to change the behavior of the operator:
+
+- **BW_API_URL** - Sets the Bitwarden API URL that the Secrets Manager SDK uses. This is useful for
+  self-host scenarios, as well as hitting European servers
+- **BW_IDENTITY_API_URL** - Sets the Bitwarden Identity service URL that the Secrets Manager SDK
+  uses. This is useful for self-host scenarios, as well as hitting European servers
+- **BW_SECRETS_MANAGER_STATE_PATH** - Sets the base path where Secrets Manager SDK stores its state
+  files
+- **BW_SECRETS_MANAGER_REFRESH_INTERVAL** - Specifies the refresh interval in seconds for syncing
+  secrets between Secrets Manager and K8s secrets. The minimum value is 180.
+
+## Running and Debugging
+
+1. Install the Custom Resource Definition into the cluster using `make install` or by using the
+   Visual Studio Task called "apply-crd" from the "Tasks: Run Task" in the command palette.
+
+1. To debug the code, just hit F5. You can also use `make run` at the command line to run without
```

Comment:
ℹ️ While Markdown is fine with both, we often increment the numbered lists in these docs.

Reply:
VS Code Markdown editor doesn't love that.

Reply:
I went ahead and updated this.